programs/survival/README.md

```diff
@@ -97,7 +97,7 @@ Various scripts that modify various game elements, often replicating popular mod
 #### By gnembonmc
 	There is a video on his channel about this.
 	If you right click with an empty hand on a bunch of stained glass, it will disappear for a moment and then reappear.
-	(Please input more detailed explanation)
+	Its basically like an automated glass sliding door but insteading of sliding the door disappears.
```

Comment:
I'd not add the `basically`, IMO it makes it sound less cool than it seemed when it was showcased.

Reply:
Also not sure if is this more detailed...

Reply:
indeed...

Reply:
Made the suggested changes. Thanks